BryantUniversity/BryantUniversity/Controllers/FacultyController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using BryantUniversity.Data;
+using BryantUniversity.Models;
+using BryantUniversity.Models.Repo;
+using BryantUniversity.ViewModels;
+using System.Web.Mvc;
+
+namespace BryantUniversity.Controllers
+{
+    public class FacultyController : Controller
```

Comment:
Not sure if FacultyController is the appropriate name for creating all other types of accounts, I think we can just use Account Controller and add Action Methods there